src/theory/bags/bags_rewriter.cpp

```diff
@@ -654,6 +655,20 @@ BagsRewriteResponse BagsRewriter::postRewriteFold(const TNode& n) const
   }
   return BagsRewriteResponse(n, Rewrite::NONE);
 }
+
+BagsRewriteResponse BagsRewriter::postRewriteProduct(const TNode& n) const
```

Comment:
I'd expect there to be a rewrite that does evaluation in the case that all arguments are constant.

Reply:
Yes, this function is only called if not all children are constants. 
When they are constants, this function
https://github.com/cvc5/cvc5/blob/master/src/theory/bags/bags_rewriter.cpp#L68
will call `evaluateProduct` which computes the product from constants. 
https://github.com/cvc5/cvc5/pull/8020/files#diff-a9026cb0a61c5f5cc41cb5b0753617a74d133dd35186dc08fa37d192df4f69a2R119

Reply:
Ah cool, didnt realize this was handled more generally.